rbenchmarker/benchmarker/test.yaml

```diff
@@ -0,0 +1,18 @@
+# Where to get the source data.
+# Each data source has multiple data formats with their file paths. The format tells us what function to use to
+# convert the source data into a common data structure (i.e DelayedArray). The args optionally
+# tell us some arguments to pass to that function (i.e. '{format}_to_delayedarray' defined in file_to_delayedarray.R).
+#file_location: local
+sources:
+  tenx_mouse_neuron_20k:
+    h5_1k_1k:
+      path: hdf5_1000_1000_chunks_3_compression.h5
```

Comment:
Could be helpful to note where you got this file.

Reply:
added